### ISSUE TITLE
Update project layout and instructions to simplify Community setup

### ISSUE THREAD

**Issue by muenzpraeger** [trailheadapps/redwoods-insurance#91]
**What is missing from the application? Please describe.**
The current setup involves many manual steps to get the demo up and running. This can be challenging for new users.

**Describe the solution you'd like**
The project set up should be simplified using community metadata in source, and the newer Salesforce CLI commands around communities, similar to [eBikes](https://github.com/trailheadapps/ebikes-lwc).

**Comment by muenzpraeger** [comment]
Why is this closed?

**Comment by codefriar** [comment]
I closed the wrong issue.

**Comment by codefriar** [comment]
@muenzpraeger worked on this yesterday. The experience bundle apis are currently throwing gacks. Genius Mathews has been notified but I’ve not heard back.

**Comment by muenzpraeger** [comment]
`ExperienceBundle` is just one element. There are also other steps that can be simplified using Salesforce CLI, like user creation, community creation and publish etc.

**Comment by codefriar** [comment]
@muenzpraeger two things.

1. when summer20 is merged, this will be resolved. The steps have been simplified by using experience bundles (which have been fixed) and new sfdx commands just like ebikes. 
2. Unfortunately, this is an *authenticated* community, and users will still have many manual steps that we cannot, to my knowledge, automate. Unfortunately, this is as far as I got before deadline for summer20. Look for other issues to further refine this.
